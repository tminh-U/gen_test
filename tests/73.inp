10
4 20 18 1 12 14 1 13 1 8 
10
7 10
6 6
6 9
1 4
3 9
1 1
4 10
8 8
4 10
6 10
