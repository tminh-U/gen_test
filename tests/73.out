10
2
7
7
21
2
28
2
28
11
